.circleci/continue_config.yml

```diff
@@ -2841,11 +2841,9 @@ jobs:
             make -j $(nproc) ddappsec_helper_test
             ./tests/helper/ddappsec_helper_test
       - run:
-          name: Download ruleset
+          name: Copy ruleset
           command: |
-            cd appsec/build
-            make event_rules
-            cp ./third_party/event_rules-prefix/src/event_rules/build/recommended.json ../../appsec_$(uname -m)/
+            cp recommended.json ../appsec_$(uname -m)/
```

Comment:
Looks like something is wrong with the path here

Reply:
Yes! I've just fixed it